polytope/polytope.py

```diff
@@ -291,26 +291,34 @@ def intersect(self, other, abs_tol=ABS_TOL):
         return reduce(Polytope(iA, ib), abs_tol=abs_tol)
 
     def translation(self, d):
-        """Translate the Polytope by a given vector.
+        """Returns a copy of C{self} translated by the vector C{d}.
 
         Consult L{polytope.polytope._translate} for implementation details.
 
         @type d: 1d array
-        @param d: The translation vector.
-
-        @rtype: L{Polytope}
-        @return: A translated copy of the Polytope.
         """
         newpoly = self.copy()
         _translate(newpoly, d)
         return newpoly
 
     def rotation(self, u, v, theta=None):
-        """Rotate the Polytope. Only simple rotations are implemented at this
+        """Returns a rotated copy of C{self}.
+
+        Use one of two methods that to describe the plane of rotation and the
+        angle of rotation (in radians) with u, v, and theta.
+
+        (1) u and v are the indicies (0, N] of two basis vectors, and theta is
```

Comment:
Special cases aren't special enough to break the rules [PEP 20]. It seems more uniform to let `u, v` be vectors. This way:

1. the interface remains unchanged when the implementation is extended to handle arbitrary vectors
2. an `UnimplementedError` is raised if vectors outside the intended basis are passed
3. users don't have to think about any particular basis
4. index-based thinking is avoided. This is more Pythonic and more geometric / structural.

Defining the basis vectors `u0, u1` etc in user code is relatively simple, and it also names them, so it is probably easier to read (as opposed to some integers 0, 1, ...).